pkg/http/token_test.go

```diff
@@ -0,0 +1,236 @@
+package http_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.homedepot.com/cd/arcade/pkg/google/googlefakes"
+	arcade_http "github.homedepot.com/cd/arcade/pkg/http"
```

Comment:
I can't find the source on this right now, but I don't think we should be putting underscores on our imported packages. Instead maybe just say `arcadehttp`.

Reply:
Well I found the source that is referencing package names. I usually follow this convention for import names as well. https://blog.golang.org/package-names